src/frontend/src/binder/values.rs

```diff
@@ -80,6 +81,25 @@ impl Binder {
             schema,
         })
     }
+
+    /// Bind row to `struct_value` for nested column,
+    /// e.g. Row(1,2,(1,2,3)).
+    /// Only accept value and row expr in row.
+    pub fn bind_row(&mut self, exprs: &[Expr]) -> Result<StructValue> {
+        let fields = exprs
+            .iter()
+            .map(|e| match e {
+                Expr::Value(value) => Ok(self.bind_value(value.clone())?.get_data().clone()),
+                Expr::Row(expr) => Ok(Some(self.bind_row(expr)?.to_scalar_value())),
+                _ => Err(ErrorCode::NotImplemented(
+                    format!("unsupported expression {:?}", e),
+                    112.into(),
```

Comment:
What do you mean https://github.com/singularity-data/risingwave/issues/112? Are there more exprs remaining to be supported in Row()?

Reply:
My fault, I misunderstand that 112 use to all unsupported expression, I change this to None.